OneCommunityGlobal/HGNRest
# Equipment_List_View

This PR is about the Equipments List View from Phase 2 with Sorting based on project name and item name, Filtering based on selected values of project and equipment, and Modal implementation for viewing update and purchase history.
Implements # (WBS) 9.1.2

Related PRS (if any):
To test this backend PR you need to check out the #1937  frontend PR.

Main changes explained:
Created a fetch API for equipments

How to test:
1. check into the current branch
2. do npm run build and npm run start to run this PR locally
3. Test api : http://localhost:4500/api/bm/equipments
